ngraph/core/src/op/util/attr_types.cpp

```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 //*****************************************************************************
 #include <map>
+#include <cctype>
```

Comment:
looks like this header should be before `<map>` (this is needed to pass clang code-style check, where headers are located in lexicographical order)

Reply:
Fixed, thank you!